changelog/_unreleased/2024-11-28-improved-checkout-for-a11y.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Improved Checkout accessibility
+issue: NEXT-38513
+---
+# Core
+* Changed `Shopware\Core\Checkout\Payment\PaymentMethodCollection` to not re-sort selected payment method.
+* Changed `Shopware\Core\Checkout\Shipping\ShippingMethodCollection` to not re-sort selected shipping method.
+___
+# Storefront
+* Changed `Resources/views/storefront/component/payment/payment-fields.html.twig` to list all payment methods instead of only 5.
+* Changed `Resources/views/storefront/component/payment/payment-method.html.twig` to show the payment method description only for selected payment method.
+* Changed `Resources/views/storefront/component/shipping/shipping-fields.html.twig` to list all shipping methods instead of only 5.
+* Changed `Resources/views/storefront/component/shipping/shipping-method.html.twig` to show the shipping method description only for selected shipping method.
+* Deprecated `Resources/views/storefront/component/payment/payment-fields.html.twig` to remove collapse logic, payment method list is now in `Resources/views/storefront/component/payment/payment-form.html.twig`
+* Deprecated `Resources/views/storefront/component/shipping/shipping-fields.html.twig` to remove collapse logic, payment method list is now in `Resources/views/storefront/component/shipping/shipping-form.html.twig`
+* Deprecated blocks for rename:
```

Comment:
please also fill the major change section with this so everyone has an easier time migrating

Reply:
we don't list deprecated blocks there, only removed templates, which I now did.